### PR TITLE
Implemented AudioManager 

### PR DESCRIPTION
**Pull request by prene112**

### CODE REVIEW

File: Assets/Game/GUI/Scripts/MenuSystem/MenuNavigator.cs [hunk 1]
```diff
@@ -28,35 +28,40 @@ protected MenuNode Current
     void Start()
     {
         Current = start;
-        InputMap.Instance.inputEnabled = true;
```
Comment (BrandonShockley) [inline Assets/Game/GUI/Scripts/MenuSystem/MenuNavigator.cs]: Why did you remove this?

Comment (prene112) [reply Assets/Game/GUI/Scripts/MenuSystem/MenuNavigator.cs]: i think it was an accident i will put it back
